exchanges/exchange.go

```diff
@@ -671,28 +671,30 @@ func (b *Base) UpdatePairs(incoming currency.Pairs, a asset.Item, enabled, force
 			updateType = "available"
 		}
 
-		if force {
-			log.Debugf(log.ExchangeSys,
-				"%s forced update of %s [%v] pairs.",
-				b.Name,
-				updateType,
-				strings.ToUpper(a.String()))
-		} else {
-			if len(diff.New) > 0 {
-				log.Debugf(log.ExchangeSys,
-					"%s Updating %s pairs [%v] - Added: %s.\n",
-					b.Name,
-					updateType,
-					strings.ToUpper(a.String()),
-					diff.New)
-			}
-			if len(diff.Remove) > 0 {
+		if b.Verbose {
+			if force {
```

Comment:
🥤 Force seems weird and seems like it can be rm'd but definitely not for this PR.

Reply:
Was used when we needed to force update pairs when we did config upgrades per exchange but that code has all been removed now, so should  be safe to remove